lib/commands/deploy/index.js

```diff
@@ -38,13 +38,17 @@ class DeployCommand extends AbstractCommand {
         try {
             profile = profileHelper.runtimeProfile(cmd.profile);
             new ResourcesConfig(path.join(process.cwd(), CONSTANTS.FILE_PATH.ASK_RESOURCES_JSON_CONFIG));
+            Messenger.getInstance().info(`Deploy configuration loaded from ${CONSTANTS.FILE_PATH.ASK_RESOURCES_JSON_CONFIG}`);
+            helper.confirmProfile(profile);
             this._filterAlexaHostedSkill(profile);
             this._initiateManifestModel(profile);
         } catch (err) {
             if (err instanceof CliWarn) {
                 Messenger.getInstance().warn(err.message);
             } else if (err instanceof CliFileNotFoundError) {
                 Messenger.getInstance().warn(err.message);
+            } else if (err instanceof CliError) {
```

Comment:
3 IFs can be collapsed to one `if (err instanceof CliError)` . CliWarn and CliFileNotFoundError extend CliError

Reply:
But the log level is different. The first two are about warning message, the last one is error. That's why I check CliError class at the last step. Do you think this makes sense?

Reply:
If the final result is the same - command fails, I think it should be handled the same way. Up to you.